libs/frontend/ui-widgets/src/lib/form/PSMultiselect.vue

```diff
@@ -23,7 +23,7 @@
         autocomplete="off"
         @update:modelValue="updateSearch"
         @keydown.enter.stop="searchEnter"
-        @keydown.enter.tab="searchEnter"
+        @keydown.tab.stop="searchEnter"
```

Comment:
Fixed a bug where select event will be emitted twice when enter is pressed.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/3ef276aa-afff-47cc-ba76-51e8a0930a25

Reply:
There's still some odd behavior with multiselect, but I think I can fix it in #147